app/crud.py

```diff
@@ -0,0 +1,85 @@
+from sqlalchemy.orm import Session
+from . import models, schemas
+from datetime import datetime
+
+
+def create_book(db: Session, book: schemas.BookCreate, library_id: int):
+    book_data = book.dict()
+    book_data.pop("library_id", None)
```

Comment:
We can write like this
```suggestion
   book.model_dump(exclude={"library_id"})
```